source/_posts/2024-05-09-music-assistant-2.markdown

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: "Music Assistant 2.0: Your Music, Your Players"
+description: "Connecting multiple music libraries to almost any smart speaker."
+date: 2024-05-09 00:00:01
+date_formatted: "May 09, 2024"
+author: Marcel van der Veldt
+comments: true
+categories: Music-Assistant
+og_image: /images/blog/2024-05-music-assistant/art.jpg
+---
+<p><img src='/images/blog/2024-05-music-assistant/art.jpg' class='no-shadow' /></p>
+
+Today, exactly five years ago, I, Marcel, started working on [Music Assistant](https://music-assistant.io/). What began as a quick script, to sync my playlists so I could switch between streaming providers, grew into a beast on its own.
+
+Music Assistant is what I’d like to call a “music library manager” – it gives you full control of playing YOUR music on your players. Just like Home Assistant centralizes all your devices and services to work together, Music Assistant does the same with your music sources and smart audio-playing devices. This is just one of the many [Home Assistant Add-on’s](https://www.home-assistant.io/addons/) that can massively expand its smart capabilities.
+
+{% my supervisor_addon badge addon="d5369777_music_assistant_beta" repository_url="https://github.com/music-assistant/home-assistant-addon" %}
```

Comment:
Update this to stable when Marcel releases it.

Reply:
```suggestion
{% my supervisor_addon badge addon="d5369777_music_assistant" repository_url="https://github.com/music-assistant/home-assistant-addon" %}
```

Reply:
addon is placed on the repo - that first part stays the same right (the slug) ?

Reply:
https://github.com/music-assistant/home-assistant-addon/tree/main/music_assistant